mmdet/models/dense_heads/anchor_head.py

```diff
@@ -250,8 +240,8 @@ def _get_targets_single(self,
             bbox_targets[pos_inds, :] = pos_bbox_targets
             bbox_weights[pos_inds, :] = 1.0
             if gt_labels is None:
-                # only rpn gives gt_labels as None, this time FG is 1
-                labels[pos_inds] = 1
+                # Foreground is the first class since v2.5.0
```

Comment:
\# Only rpn gives gt_labels as None. Foreground is the first class since v2.5.0.